integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/detailquery/ValueCompressionDataTypeTestCase.scala

```diff
@@ -130,7 +130,4 @@ class ValueCompressionDataTypeTestCase extends QueryTest with BeforeAndAfterAll
     file.delete()
   }
 
-  override def afterAll {
-    deleteFile(tempDirPath)
```

Comment:
why remove it?

Reply:
Because it will delete$carbondata_home/integration/spark-common-test/src/test/resources/temp/data1.csv, which will led to other test failure: org.apache.spark.carbondata.query.SubQueryTestSuite